homeassistant/helpers/device_registry.py

```diff
@@ -593,12 +593,31 @@ def async_clear_area_id(self, area_id: str) -> None:
                 self._async_update_device(dev_id, area_id=None)
 
 
-@singleton(DATA_REGISTRY)
+@bind_hass
```

Comment:
should we even bind this?

Reply:
Maybe not, removed from the new functions.